modules/gitops-release-notes-1-13-0.adoc

```diff
@@ -0,0 +1,117 @@
+
+* With this update, you customize the Keycloak route name in the {gitops-title} Operator. This update also enables support for complex routing scenarios, such as dividing routes among various ingress controller shards. link:https://issues.redhat.com/browse/GITOPS-4451[GITOPS-4451]
+
+* With this update, Argo CD CLI has been enhanced by enabling the `--app-namespace` flag for various application-related commands. This flag allows users to specify the namespace for operations such as waiting, rolling back, patching, editing, diffing, viewing history, syncing, and deleting applications. As a result, the following commands are updated in {gitops-title}: link:https://issues.redhat.com/browse/GITOPS-3834[GITOPS-3834]
+
+** `argocd app wait`
+** `argocd app rollback`
+** `argocd app patch`
+** `argocd app edit`
+** `argocd app diff`
+** `argocd app history`
+** `argocd app sync`
+** `argocd app delete`
+** `argocd app get`
+
+[id="fixed-issues-1-13-0_{context}"]
+== Fixed issues
+
+* Before this update, if the `DeploymentConfig [apps.openshift.io/v1]` API was disabled in the {OCP} cluster, the {gitops-shortname} Operator would enter the `CrashLoopBackoff` error state and would not function properly. This update fixes the issue by modifying the functionality of the {gitops-shortname} Operator to ensure that it does not enter the `CrashLoopBackoff` error state when the API is unavailable in the cluster. link:https://issues.redhat.com/browse/GITOPS-4489[GITOPS-4489]
+
+* Before this update, users were unable to connect to remote Git repositories through a SOCKS5 proxy server because it was only compatible with HTTPS URLs. With this update, users can access remote Git repositories by using a SOCKS5 proxy server that supports the SSH protocol. link:https://issues.redhat.com/browse/GITOPS-3710[GITOPS-3710]
+
+* Before this update, the `forceHttpBasicAuth` repository configuration was not initialized properly during repository updates from Argo CD Web UI. As a result, the value was reset to default, leading to connection errors, particularly for Azure DevOps repositories. This update fixes the issue by adding a change in the upstream to ensure that the `forceHttpBasicAuth` configuration is correctly initialized during repository update operations. link:https://issues.redhat.com/browse/GITOPS-3706[GITOPS-3706]
+
+* Before this update, resource deletion was not handled properly in the Argo CD upstream. With this update, Argo CD upstream is enhanced to ensure that resource deletion is managed properly. Now, Argo CD waits for resources to be entirely deleted before advancing to the next sync-wave in the deletion process. This approach ensures a controlled and predictable workflow, enhancing the user experience and preventing potential issues that might arise from not waiting for resources to be fully recycled before proceeding to the next sync-wave. link:https://issues.redhat.com/browse/GITOPS-2642[GITOPS-2642]
```

Comment:
Can we move this to new features section? and a small update to text.
```suggestion
* With this update, Argo CD upstream is enhanced to ensure that resource deletion is managed properly when using sync-waves. Now, Argo CD waits for resources to be entirely deleted before advancing to the next sync-wave in the deletion process. This approach ensures a controlled and predictable workflow, enhancing the user experience and preventing potential issues that might arise from not waiting for resources to be fully recycled before proceeding to the next sync-wave. link:https://issues.redhat.com/browse/GITOPS-2642[GITOPS-2642]
```

Reply:
Yes, I have now included this RN text in the Features section.